src/app/content/utils/seoUtils.ts

```diff
@@ -9,20 +12,168 @@ import {
   splitTitleParts,
 } from './archiveTreeUtils';
 
-const getParentPrefix = (node: LinkedArchiveTreeNode | undefined): string => {
+type PageTypes = 'page' | 'answer-key' | 'subpage' | 'eoc-page' | 'eob-page';
+
+interface DescriptionTemplateValues {
+  parentTitle: string;
+  parentType: 'chapter' | 'book' | 'other';
+  pageTitle: string;
+  bookTitle: string;
+  parentPrefix: string;
+}
+
+const domParser = new DOMParser();
+
+export const getParentPrefix = (node: LinkedArchiveTreeNode | undefined, includeTitle: boolean = false): string => {
   if (!node) {
     return '';
   }
 
   if (archiveTreeSectionIsChapter(node)) {
-    const number = getArchiveTreeSectionNumber(node);
-    return `Ch. ${number} `;
+    const number = getArchiveTreeSectionNumber(node).trim();
+    const name = getArchiveTreeSectionTitle(node);
+    return includeTitle ? `Ch. ${number}. ${name}` : `Ch. ${number} `;
   }
 
   return archiveTreeSectionIsBook(node.parent)
     ? getArchiveTreeSectionTitle(node) + ' '
-    : getParentPrefix(node.parent);
+    : getParentPrefix(node.parent, includeTitle);
+
+};
+
+const hideMath = (node: Element) => {
+  if (!node) {
+    return '';
+  }
+
+  node.querySelectorAll('math, .os-math-in-para').forEach((el: Element) => {
```

Comment:
@TomWoodward the json files that are used for testing the `receiveContent` hook don't apply the `os-math-in-para` class to math. Instead of adding another selector here like `<math>` I'm wondering if the test page markup should be updated to use the class.

Reply:
(this is all because there is a meta description test for that hook and I don't want to "expect" math gibberish)

Reply:
you could probably just update that test to mock your description util so that you don't have to worry about testing it from more than one place